controllers/k8ssandracluster_controller.go

```diff
@@ -46,6 +46,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	STARGATE_AUTH_KEYSPACE = "data_endpoint_auth"
```

Comment:
This is Java naming convention. Use StargateAuthKeyspace or equal (if it needs to be exported, otherwise of course stargateAuthKeyspace).

Reply:
Right 😅

Reply:
fixed